src/transaction/log_manager.h

```diff
@@ -247,4 +247,13 @@ void LOG_CS_PROMOTE (THREAD_ENTRY * thread_p);
 bool LOG_CS_OWN (THREAD_ENTRY * thread_p);
 bool LOG_CS_OWN_WRITE_MODE (THREAD_ENTRY * thread_p);
 
+// *INDENT-OFF*
+/* Interface used by log flush to notify threads. */
+class log_flush_lsa
```

Comment:
it seems log_flush_lsa class is never used by slave GCM. maybe it belongs to another patch?

Reply:
It is used in next patch - 23160.